lib/PnpPlugin.js

```diff
@@ -51,6 +51,19 @@ module.exports = class PnpPlugin {
 					resolution = this.pnpApi.resolveToUnqualified(packageName, issuer, {
 						considerBuiltins: false
 					});
+
+					if (resolution === null) {
+						// This is either not a PnP managed issuer or it's a Node builtin
+						// Try to continue resolving with our alternatives
+						if (resolveContext.log) {
+							resolveContext.log(`issuer is not managed by the pnpapi`);
+						}
+						// FIXME: In Webpack this stops the resolution at
+						// https://github.com/webpack/enhanced-resolve/blob/029d3e2ce802ef5181355bf64f14f6d0d819ed66/lib/ConditionalPlugin.js#L52-L53
+						// instead of continuing with the alternatives
```

Comment:
I'm not quite sure how to let it use alternatives here

Reply:
> I'm not quite sure how to let it use alternatives here

@sokra or @alexander-akait, if there's any insight into how we might be able to address a fix for this, I'd be more than happy to help! Thank you!

Reply:
@alexander-akait Thank you for approving, but the changes here won't work because of this question.